libs/@blockprotocol/type-system/crate/src/ontology/entity_type/links/mod.rs

```diff
@@ -13,7 +13,9 @@ use crate::{
 };
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct Links(HashMap<VersionedUrl, MaybeOrderedArray<Option<OneOf<EntityTypeReference>>>>);
+pub struct Links(
+    pub(crate) HashMap<VersionedUrl, MaybeOrderedArray<Option<OneOf<EntityTypeReference>>>>,
```

Comment:
quite a complex type, this looks like something clippy would usually complain about. 🤔

Reply:
I agree but I won’t change this here as it’s internal only.